src/pages/cplusplus/introduction-to-cplusplus/index.md

```diff
@@ -1,10 +1,12 @@
 ---
-title: Introduction to C++
+Introduction to C++
```

Comment:
`title: ` is required. It's used to create the title for the navigation and the title of the tab.